app/index.html

```diff
@@ -59,15 +59,15 @@
         </script>
 
         <script type="text/html" id="template-add-tree">
-            <label><input type="radio" name="type" value="tree">Tree</label>
-            <label><input type="radio" name="type" value="bush">Bush</label>
-            <label><input type="radio" name="type" value="herb">Herb</label>
-            <br>
-            <br>
-            <br>
-            <input id="description" type="text" name="description" placeholder="Description">
-            <br>
-            <br>
+            <div class="form-button">
```

Comment:
A form button should be a single element, no? Here you have wrapped 3 radio buttons and a text input field together as a button.